packages/core/strapi/lib/core/loaders/plugins/get-enabled-plugins.js

```diff
@@ -42,8 +48,13 @@ const toDetailedDeclaration = declaration => {
 };
 
 const getEnabledPlugins = async strapi => {
+  const projectDependencies = [
+    ...keys(strapi.config.get('info.dependencies', {})),
+    ...INTERNAL_PLUGINS,
```

Comment:
Should we return them at the end of the function as we don't need to check if they are actual plugins ?

Reply:
We also do other things like finding the path of the plugin in the node modules and checking it is correctly installed, so I think it's more readable to have it there no?

Reply:
Alright, Can we just make sure it's always enabled then ?